roles/oneagent/tests/README.md

```diff
@@ -27,6 +27,10 @@ $ mkdir -p roles/oneagent/files && wget https://ca.dynatrace.com/dt-root.cert.pe
 $ ansible-galaxy collection build . -vvv
 $ sudo bash -c "source venv/bin/activate && ansible-galaxy collection install -vvv dynatrace-oneagent*"
 
-# Run tests (eg. For linux_x86 platform)
+# Run tests (eg. For linux_x86 platform on local machine)
```

Comment:
please describe it more clear which platforms can be run only on localhost and which only on remote